infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/schema/impl/AbstractSszListSchema.java

```diff
@@ -75,6 +79,43 @@ protected TreeNode createTree(TreeNode dataNode, int length) {
     return BranchNode.create(dataNode, toLengthNode(length));
   }
 
+  @Override
+  public SszListT nCopies(final ElementDataT value, final int length) {
+    final int elementsPerChunk = getElementsPerChunk();
+    final TreeNode fullLeafNode;
+    TreeNode partialNode = null;
+    if (elementsPerChunk == 1) {
+      fullLeafNode = value.getBackingNode();
+    } else {
```

Comment:
Make sure that bitlist is handled specifically. 
Makes sense just to check here this is not a bitlist and override the method in `SszBitlistSchemaImpl`